lib/runtime/make.js

```diff
@@ -166,14 +188,20 @@ async function make (scope, userId, onlyInRoom) {
             }
         }, userId);
 
+        const starUsedTime = vm.isolate.nowCpuTime();
         runResult = await run.apply(undefined, [], {timeout: data.timeout});
+        const endCpuTime = vm.isolate.nowCpuTime();
 
         run.dispose();
         dataRef.dispose();
 
+        let intentCount = countIntents(runResult.intents);
+        runResult.intentsCpu = intentCount / 0.2;
```

Comment:
This is equal to a multiply with 5 (`intentCount * 5`) which doesn't seem intended.

```suggestion
        runResult.intentsCpu = intentCount * 0.2;
```